save-plugins/fix-and-warn-plugin/src/commonMain/kotlin/com/saveourtool/save/plugins/fixandwarn/FixAndWarnPluginConfig.kt

```diff
@@ -41,21 +41,10 @@ data class FixAndWarnPluginConfig(
         }
     }
 
-    override fun validateAndSetDefaults(): PluginConfig {
-        require(warn.resourceNamePattern.matches(fix.resourceNameTest) &&
```

Comment:
I think check `warn.resourceNamePattern.matches(fix.resourceNameTest)` is still required

Reply:
returned back